lib/WasmReader/WasmByteCodeGenerator.cpp

```diff
@@ -699,20 +704,17 @@ WasmBytecodeGenerator::EmitLoop()
     Js::ByteCodeLabel loopLandingPadLabel = m_writer.DefineLabel();
 
     uint loopId = m_writer.EnterLoop(loopHeadLabel);
-    PushLabel(loopTailLabel);
```

Comment:
Yielding a value to the equivalent of the `loopTailLabel` depth yields it to the enclosing block?